index.html

```diff
@@ -0,0 +1,352 @@
+  </head>
+
+  <body>
+    <header class="flex-container">
+      <div class="logo-container">
+        <a class="logo" href="">My Logo</a>
+      </div>
+      <nav class="nav_item">
+        <ion-icon class="menu" name="menu-sharp"></ion-icon>
+      </nav>
+    </header>
+
+    <section class="hero">
+      <h2>
+        I'm Samuel<br />Glad to see<br />
+        you!
+      </h2>
+      <p class="paragraph">
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+
+      <span class="connect">LETS CONNECT</span>
+
+      <div class="socialmedia_icons_container">
+        <ul class="media_icons">
+          <li>
+            <a href="#">
+              <img src="images/icons/twitter.svg" alt="Twitter Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/linkedin.png" alt="LinkedIn Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/m.svg" alt="Media Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/cat.svg" alt="Github Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/fingers.svg" alt="Victory Icon" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!--Works Section-->
+    <section class="works">
+      <div class="grid-container">
+
+        <div class="card-layout">
+          <a href="#">
+            <img
+              class="image-fluid"
+              src="images/works/image_1.png"
+              alt="background_image"
+            />
+          </a>
+          <div class="p-title">
+            <h3 class="title">Tonic</h3>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="btn-container-flex">
+              <li class="btn">
+                <button class="btn-sm" type="button">html</button>
+              </li>
+              <li class="btn">
+                <button class="btn-md" type="button">css</button>
+              </li>
+              <li class="btn">
+                <button class="btn-lg" type="button">javaScript</button>
+              </li>
+            </ul>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
```

Comment:
- [ ] Great job so far but I just have some notes to make your website nicer, I think here you need to add some margin/padding between the heading and the paragraph to make it similar to the Figma template 

- [ ] I think it would be great if you used the same color for the technologies `html` , `css` and `javascript`

- [ ] I think it would be nicer if you add some font weight on the color of the button for see project to make it very simial to the Figma as required 👌🏻💯

> I will leave an image to what you need to fix here 💯🚀🔥
![image](https://user-images.githubusercontent.com/58553711/185278071-dd75c4aa-ca7b-4c32-be8c-a521e31488bd.png)

> Figma Template
![image](https://user-images.githubusercontent.com/58553711/185278151-02a749d3-55bb-44c0-b38e-b32e39ab8724.png)